sources/packages/api/src/route-controllers/institution-locations/institution-location.institutions.controller.ts

```diff
@@ -246,6 +259,80 @@ export class InstitutionLocationInstitutionsController extends BaseController {
       applicationOfferingName: offering.name,
       applicationProgramDescription: offering.educationProgram.description,
       applicationProgramName: offering.educationProgram.name,
+      applicationProgramCredential: credentialTypeToDisplay(
+        offering.educationProgram.credentialType,
+      ),
+      applicationProgramDelivery: deliveryMethod(
+        offering.educationProgram.deliveredOnline,
+        offering.educationProgram.deliveredOnSite,
+      ),
+      applicationOfferingStudyDelivery: offering.offeringDelivered,
+      applicationOfferingStudyBreak: offering.studyBreaks.map((studyBreak) => ({
+        breakStartDate: dateString(studyBreak.breakStartDate),
+        breakEndDate: dateString(studyBreak.breakEndDate),
+      })),
+      applicationOfferingTuition: offering.actualTuitionCosts,
+      applicationOfferingProgramRelatedCosts: offering.programRelatedCosts,
+      applicationOfferingMandatoryFess: offering.mandatoryFees,
+      applicationOfferingExceptionalExpenses: offering.exceptionalExpenses,
     };
   }
+
+  /**
+   * Save scholastic standing and create new assessment.
+   * @param locationId location id to check whether the requested user and the requested application has the permission to this location.
+   * @param applicationId application id.
+   * @UserToken institution user token
+   * @param payload Scholastic Standing payload.
+   */
+  @ApiBadRequestResponse({ description: "Invalid form data." })
+  @ApiUnprocessableEntityResponse({
+    description:
+      "Application not found or invalid application or invalid" +
+      " application status or another assessment already in progress.",
+  })
+  @HasLocationAccess("locationId")
+  @Post(":locationId/application/:applicationId/scholastic-standing")
+  async saveScholasticStanding(
+    @Param("locationId") locationId: number,
+    @Param("applicationId") applicationId: number,
+    @Body() payload: ScholasticStandingAPIInDTO,
+    @UserToken() userToken: IInstitutionUserToken,
+  ): Promise<void> {
+    try {
+      const submissionResult = await this.formService.dryRunSubmission(
+        FormNames.ReportScholasticStandingChange,
+        payload.data,
+      );
+
+      if (!submissionResult.valid) {
+        throw new BadRequestException("Invalid submission.");
+      }
+      const scholasticStanding =
+        await this.studentScholasticStandingsService.saveScholasticStandingCreateReassessment(
+          locationId,
+          applicationId,
+          userToken.userId,
+          submissionResult.data.data,
+        );
+
+      // Start assessment.
+      if (scholasticStanding.studentAssessment) {
+        await this.studentAssessmentService.startAssessment(
+          scholasticStanding.studentAssessment.id,
+        );
+      }
+    } catch (error) {
```

Comment:
Sorry, I missed this before, but the error here can be typed as `error: unknown` as we talked 😉